src/higlass/_utils.py

```diff
@@ -75,7 +76,7 @@ def ensure_list(x: T | list[T] | None) -> list[T]:
 
 def copy_unique(model: ModelT) -> ModelT:
     """Creates a deep copy of a pydantic BaseModel with new UID."""
-    copy = model.__class__(**model.model_dump())
+    copy = deepcopy(model)
```

Comment:
Not familiar enough to know the consequences of `deepcopy` on a pydantic model, but pydantic appears to have this API:
```suggestion
    copy = model.copy(deep=True)
```

Reply:
So that works too. I don't know what the difference is either but I switched over to `model.model_copy (deep=True)` (`model.copy` is deprecated).